packages/schema/src/schema/project/layers/symbology/vectorColor.json

```diff
@@ -10,7 +10,6 @@
         },
         "selectedAttribute": {
             "type": "string",
-            "title": "Attribute",
```

Comment:
I wanted to use this for form generation, but it's also affecting type generation.